aladdin-container.sh

```diff
@@ -85,6 +85,11 @@ function environment_init() {
     echo "CLUSTER_CODE = $CLUSTER_CODE"
     echo "NAMESPACE = $NAMESPACE"
 
+    # Copy our .ssh directory over. This is a special case due to
+    # SSH permision and ownership checks on the ~/.ssh files.
+    rm -rf $HOME/.ssh
+    cp -r $HOME/_ssh $HOME/.ssh
```

Comment:
During my testing, non-`IS_LOCAL` invocations don't get into that `if` statement below, so I had to put this out here.